tests/integration/standard/test_metrics.py

```diff
@@ -143,6 +143,8 @@ def test_unavailable(self):
         self.assertTrue(results)
 
         # Stop node gracefully
+        # Sometimes this commands continues with the other nodes having not noticed
+        # 1 is down, and a Timeout error is returned instead of an Unavailable
```

Comment:
Yeah, I'm not sure if it's something with the environment or something with CCM, I'll have to look a bit more into that, although I don't think it's something we should worry about.